language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/completion/MethodCallExpressionNodeContextTest.java

```diff
@@ -0,0 +1,30 @@
+package org.ballerinalang.langserver.completion;
+
+import org.ballerinalang.langserver.commons.workspace.WorkspaceDocumentException;
```

Comment:
Do we need this new TestClass? We already have method call expression test cases in the expression_context tests. Let's move these there or vise-versa.

Reply:
I'll move it there. It is better.